resources/views/site/home.blade.php

```diff
@@ -105,71 +106,109 @@ function toggleActive () {
+                        <li class="nav-item">
+                            <a class="nav-link" href="{{ route('site.atc.newController') }}">Become a Controller</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="https://cts.vatsim.uk/home/solo.php">Solo Endorsements</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="https://cts.vatsim.uk/home/validations.php">Special
+                                Endorsements</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link"
+                               href="https://community.vatsim.uk/files/downloads/category/4-policy-documents/">Regulations
+                                and Policies</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="{{ route('visiting.landing') }}">Visit / Transfer</a>
+                        </li>
+                    </ul>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="#">Community <span class="arrow"></span></a>
+                    <ul class="nav">
+                        <li class="nav-item">
+                            <a class="nav-link" href="https://community.vatsim.uk">Forum</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="{{ route('site.community.teamspeak') }}">TeamSpeak / Slack</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="https://www.facebook.com/vatsimuk">Facebook</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="https://www.twitter.com/vatsimuk">Twitter</a>
+                        </li>
+                    </ul>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="#">Events <span class="arrow"></span></a>
+                    <ul class="nav">
+                        <li class="nav-item">
+                            <a class="nav-link" href="https://cts.vatsim.uk/bookings/calendar.php">Calendar</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="{{ route('site.marketing.live-streaming') }}">Live Streams</a>
+                        </li>
+                    </ul>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="{{ route('mship.feedback.new') }}">Feedback</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="#">Contact Us <span class="arrow"></span></a>
```

Comment:
Does this need to be a dropdown? Surely we can just direct them to the Helpdesk straight from the home page.